apps/frontend/src/app/3_organisms/RuneBridgeDialog/components/RSKToRune/RSKToRune.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useCallback } from 'react';
+
+import { SendFlowStep } from '../../contexts/sendflow';
+import { useSendFlowService } from '../../hooks/useSendFlowService';
+import { GoBackButton } from '../GoBackButton';
+import { AddressForm } from './components/AddressForm';
+import { AmountForm } from './components/AmountForm';
+import { ConfirmationScreens } from './components/ConfirmationScreens';
+import { MainScreen } from './components/MainScreen';
+
+interface RSKToRuneProps {
```

Comment:
```suggestion
type RskToRuneProps {
```
We use `type` instead of `interface` if possible.